packages/node_modules/@ciscospark/widget-recents/src/container.js

```diff
@@ -736,26 +414,20 @@ RecentsWidget.propTypes = {
 
 RecentsWidget.defaultProps = {...defaultProps};
 
-export default connect(
-  getRecentsWidgetProps,
-  (dispatch) => bindActionCreators({
-    addError,
-    checkWebRTCSupport,
-    connectToMercury,
-    declineIncomingCall,
-    dismissIncomingCall,
-    fetchAvatar,
-    fetchSpace,
-    fetchSpaces,
-    fetchTeams,
-    getFeature,
-    removeError,
-    removeSpace,
-    updateSpaceRead,
-    updateSpaceWithActivity,
-    storeActivities,
-    updateSpaceKeywordFilter,
-    updateSpaceTypeFilter,
-    updateWidgetStatus
-  }, dispatch)
+export default compose(
```

Comment:
removed all the redux actions that were no longer needed or moved to the enhancer.